pages/index.js

```diff
@@ -144,12 +159,13 @@ const FormContainer = styled.form`
   }
 
   button {
```

Comment:
You can't nest a button in a Button Component. This contradict the semantic html.